### PR TITLE
Clarify SRD() won't reject on identity failure wo/target peer identity.

### PR DESCRIPTION
**Pull request by jan-ivar**
Fixes https://github.com/w3c/webrtc-pc/issues/2148.

cc @martinthomson

***
<a href="https://pr-preview.s3.amazonaws.com/jan-ivar/webrtc-pc/pull/2250.html" title="Last updated on Aug 30, 2019, 7:07 PM UTC (c1d194a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-pc/2250/f665b4d...jan-ivar:c1d194a.html" title="Last updated on Aug 30, 2019, 7:07 PM UTC (c1d194a)">Diff</a>

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -3273,10 +3273,13 @@ <h2>Methods</h2>
                 promise returned <code><a>setRemoteDescription</a></code> is <a>resolved</a>.
                 If <a href="#dfn-validate-the-identity">identity validation</a> fails,
                 then the promise returned by <code><a>setRemoteDescription</a></code>
-                is <a>rejected</a>.</p>
+                MUST be <a>rejected</a>.</p>
                 <p>If there is no <a>target peer identity</a>, then
-                <code>setRemoteDescription</code> does not await the completion
-                of identity validation.</p>
+                <code>setRemoteDescription</code> MUST not await the completion
```
Comment (henbos) [inline webrtc.html]: MUST NOT

File: webrtc.html [hunk 2]
```diff
@@ -3273,10 +3273,13 @@ <h2>Methods</h2>
...
+                of identity validation. Whether
+                <a href="#dfn-validate-the-identity">identity validation</a> fails
+                or not MUST NOT influence whether the promise returned by
+                <code><a>setRemoteDescription</a></code> is rejected.</p>
```
Comment (henbos) [inline webrtc.html]: These two sentences seem to directly contradict each other?
"If identity validation fails, then the promise returned by setRemoteDescription MUST be rejected."
"Whether identity validation fails or not MUST NOT influence whether the promise returned by setRemoteDescription is rejected."

Comment (alvestrand) [reply webrtc.html]: I think that "If there is no target peer identity" on line 3277 is intended as the "else" branch of "If a target peer identity is set" on line 3271, leaving the seemingly conflicting statements on two different branches.

Is there a better way to format this?

**Comment by henbos** [comment]
Can you break things out to make it clear that we are talking about the case of not having a peer identity?

**Comment by jan-ivar** [comment]
@henbos please see if it reads better now.

**Pull request merged**